py/server/deephaven/stream/kafka/consumer.py

```diff
@@ -184,17 +247,28 @@ def consume(
             raise ValueError("at least one argument for 'key' or 'value' must be different from KeyValueSpec.IGNORE")
 
         kafka_config = j_properties(kafka_config)
-        return Table(
-            j_table=_JKafkaTools.consumeToTable(
+        if not to_partitioned:
```

Comment:
Thoughts on:

```python
java_func = _JKafkaTools.consumeToTable if not to_partitioned else _JKafkaTools.consumeToPartitionedTable
return java_func(kafka_config, topic, ...)
```
and then having the calling methods wrap w/ Table/PartitionedTable?

Reply:
That would make the code a bit cleaner and more concise, but OTH we won't be able property type hint the return type, just more a generic jpy.JType. I slightly prefer to keep it as it.

Reply:
I'll approve as-is; I personally am not too concerned about type hinting for *internal* methods in this case. An alternative is also:

```python
java_func = _JKafkaTools.consumeToTable if not to_partitioned else _JKafkaTools.consumeToPartitionedTable
val = java_func(kafka_config, topic, ...)
return Table(val) if not to_partitioned else PartitionTable(val)
```

Feel free to take the approach your most prefer.